Source/WebCore/html/HTMLMetaElement.h

```diff
@@ -50,11 +50,15 @@ class HTMLMetaElement final : public HTMLElement {
     void didFinishInsertingNode();
     void removedFromAncestor(RemovalType, ContainerNode&) final;
 
-    void process();
+    void process(const AtomString* oldValue = nullptr);
```

Comment:
`const AtomString& oldValue = nullAtom()` then you can check `oldValue.isNull()` in the implementation. AtomString has a null state so we don't need to use a raw pointer here.